Android/app/src/main/java/io/github/project_travel_mate/login/LoginActivity.java

```diff
@@ -233,4 +245,37 @@ public void showMessage(String message) {
         Toast.makeText(this, message, Toast.LENGTH_LONG)
                 .show();
     }
+
+    public boolean checkEmail(String email){
+        Matcher matcher = Patterns.EMAIL_ADDRESS.matcher(email);
+        if (!email.equals("") && matcher.matches()){
+            return true;
+        } else{
+            Toast.makeText(this, R.string.invalid_email,
+                    Toast.LENGTH_SHORT).show();
+            return false;
+        }
+    }
+    public boolean checkPassword(String passString){
```

Comment:
change `checkPassword` and `checkEmail` to
`validateEmail` and `validatePassword`